### PR TITLE
Add some more logging

### PR DESCRIPTION
**Pull request by estrattonbailey**

### CODE REVIEW

File: src/state/session/index.tsx [hunk 1]
```diff
@@ -181,14 +181,10 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
...
-        {
-          service,
-          handle,
-        },
-        logger.DebugContext.session,
-      )
+      logger.info(`session: creating account`, {
+        service,
+        handle,
+      })
```
Comment (estrattonbailey) [inline src/state/session/index.tsx:187]: Just makes sure this gets sent to Sentry prior to a thrown exception (handled in create account state reducer)

File: src/view/com/auth/create/state.ts [hunk 2]
```diff
@@ -136,7 +136,13 @@ export async function submit({
         msg`Invite code not accepted. Check that you input it correctly and try again.`,
       )
     }
-    logger.error('Failed to create account', {error: e})
+
+    if ([400, 429].includes(e.status)) {
```
Comment (estrattonbailey) [inline src/view/com/auth/create/state.ts:140]: Kinda basic, but should give us a place to go from here

File: src/view/com/auth/login/LoginForm.tsx [hunk 3]
```diff
@@ -107,17 +107,21 @@ export const LoginForm = ({
...
         setError(_(msg`Invalid username or password`))
       } else if (isNetworkError(e)) {
+        logger.warn('Failed to login due to network error', {error: errMsg})
         setError(
           _(
             msg`Unable to contact your service. Please check your Internet connection.`,
           ),
         )
       } else {
+        logger.warn('Failed to login', {error: errMsg})
```
Comment (estrattonbailey) [inline src/view/com/auth/login/LoginForm.tsx:124]: This is the existing event, so unknown errors will continue to accrue here. Numbers should go down.

File: src/view/com/auth/create/state.ts [hunk 4]
```diff
@@ -136,7 +136,13 @@ export async function submit({
...
+      logger.warn('Failed to create account', {error: e})
```
Comment (estrattonbailey) [inline src/view/com/auth/create/state.ts:141]: Most of these are just validation errors. We should look out for spikes though.

**Review by pfrazee — APPROVED** [review]
Yeah good stuff

**Pull request merged**